system-tests/lib/dep-installer/index.ts

```diff
@@ -288,11 +288,14 @@ export async function scaffoldCommonNodeModules () {
     '@cypress/webpack-dev-server',
     '@packages/socket',
     '@packages/ts',
+    '@packages/v8-snapshot-require',
     '@tooling/system-tests',
+    '@tooling/v8-snapshot',
     'bluebird',
     'chai',
     'dayjs',
     'debug',
+    'electron',
```

Comment:
As the comment mentions, if these modules are only needed by the `v8-snapshot/*` projects, they shouldn't be added to this list. Can they be removed? 

I'm guessing this was done to avoid adding install time and avoid duplication in the example project `package.json`s. To get around that, you could export `symlinkNodeModule` from this file and use it in the tests, like in `onRun`.

Reply:
Doh. Good catch. Updated here: [`9ac5768` (#23566)](https://github.com/cypress-io/cypress/pull/23566/commits/9ac57680068b11b55ee9c565d3df2f0c80892f0a)